test/Backdoor.h

```diff
@@ -1,9 +1,10 @@
-#ifndef SAILGAME_TEXAS_BACKDOOR
-#define SAILGAME_TEXAS_BACKDOOR
+#pragma once
 
 #include <src/Dummy.h>
 #include <src/defines.h>
 
+namespace SailGame {
+namespace Texas {
```

Comment:
How about put this into namespace `Test` as well?

Reply:
it is a friend class, the namespace must be same.

Reply:
ok then